src/app/child-dev-project/schools/children-overview/children-overview.component.html

```diff
@@ -0,0 +1,19 @@
+<button
+  mat-stroked-button
+  color="accent"
+  (click)="addChildClick()"
+>
+  <mat-icon style="font-size: 1.49em;"
+    class="button-icon"
+    aria-label="add child"
+    fontIcon="fa-plus-circle"
+  ></mat-icon>
```

Comment:
The styling of the '+' button is a little off

![add_child_button](https://user-images.githubusercontent.com/33730997/129026514-9cecefd0-7c40-4690-a426-949cffe3e104.png)